frontend/src/pages/profile-settings/profile-settings.tsx

```diff
@@ -0,0 +1,28 @@
+import { Switch } from '#libs/components/components.js';
+import { useAppForm } from '#libs/hooks/hooks.js';
+
+import { Setting } from './components/components.js';
+import styles from './styles.module.scss';
+
+const ProfileSettings: React.FC = () => {
+  const { control } = useAppForm({ defaultValues: { notification: true } });
```

Comment:
I think you should move default values in libs/constants.ts as it's done for sign up for example

Reply:
I was thinking about this, but not sure. Let's also ask @Liza-Veis, @v0ldemar01, @fshabanov

Reply:
I agree with @danilittaa, move default values to a separate file

Reply:
no problem